packages/product-react/src/components/Tabs/TabView.scss

```diff
@@ -0,0 +1,3 @@
+.ray-product__tab-view:focus {
+  outline: none;
```

Comment:
I know people are going to ask about this and I know that this (http://www.outlinenone.com/) exists... But I stand by NOT having the blue accessibility outline around the TabView.

Reply:
I kind of like the outline for tabbing, but not for mouse clicks.

Reply:
Oh yeah I'm good on the tabs, but now that tabview itself :-P